src/ruby/lib/grpc/generic/active_call.rb

```diff
@@ -226,6 +226,23 @@ def send_status(code = OK, details = '', assert_finished = false,
       nil
     end
 
+    def server_unary_response(req, marshalled: false, trailing_metadata: {},
+                              code: Core::StatusCodes::OK, details: 'OK')
```

Comment:
What is the point of the `marshalled` argument? I don't see any changes that use it, and I wouldn't expect users to pre-marshal data before passing it to us.